src/servant/tt_001/F84.c

```diff
@@ -14,7 +14,46 @@ INCLUDE_ASM("servant/tt_001/nonmatchings/F84", func_us_80171284);
 
 void func_us_80171560(void) {}
 
-INCLUDE_ASM("servant/tt_001/nonmatchings/F84", func_us_80171568);
+#if defined(VERSION_PSP)
+void func_psp_092EA460(Entity* self, s32 entityId, s32 params) {
```

Comment:
I know this function name can be mapped via symbols later on, but I thought it might be good to explicitly ifdef this because of the different return type and param count